.github/workflows/build.yml

```diff
@@ -137,7 +137,7 @@ jobs:
           [[ ${{ steps.test_env.outputs.type }} != first ]] && sudo ./ci/init-${{ steps.test_env.outputs.type }}-test-service.sh after
           echo "Linux launch services, done."
       - name: Start Dubbo Backend
-        if: matrix.os_name == 'linux_openresty' && steps.test_env.outputs.type == 'plugin'
+        if: matrix.os_name == 'linux_openresty' && [steps.test_env.outputs.type == 'plugin' || steps.test_env.outputs.type == 'xrpc']
```

Comment:
No need to add this

Reply:
https://github.com/apache/apisix/actions/runs/5751049180
Incorrect expression caused the CI to skip, please fix it.

Reply:
ok

Reply:
It still failed, please check the syntax

Reply:
![image](https://github.com/apache/apisix/assets/9354193/1303d133-6ef6-450c-9ba7-e8940e3bfef0)